File: model/src/main/java/no/nav/pensjon/vtp/testmodell/kodeverk/Rolle.java
package no.nav.pensjon.vtp.testmodell.kodeverk;

public enum Rolle {
    EKTE, BARN, FARA(true), MORA(true), SAMB, REPA, MMOR(true);

    private boolean forelder;

    Rolle() {
    }

    Rolle(boolean forelder) {
        this.forelder = forelder;
    }

    public boolean erForelder() {
        return forelder;
    }
}

File: vtp-pensjon-application/src/main/kotlin/no/nav/pensjon/vtp/testmodell/kodeverk/Rolle.kt
package no.nav.pensjon.vtp.testmodell.kodeverk

@Suppress("unused")
enum class Rolle(val forelder: Boolean = false) {
    EKTE,
    BARN,
    FARA(true),
    MORA(true),
    SAMB,
    REPA,
    MMOR(true);

    fun erForelder(): Boolean {
        return forelder
    }
}
